css/main.css

```diff
@@ -0,0 +1,42 @@
+body {
+    background-color: #ffffff;
+  }
+
+  h1 {
+    text-align: center;
+  }
+
+  .row {
+    padding: 0px;
+  }
+
+  .controls {
+    text-indent: 40px;
+    padding: 20px;
+  }
+
+  .form-control {
+    border:1px solid #555555;
+    border-radius: 0px;
+  }
+
+  .form-horizontal {
+    margin-left: 40px;
+  }
+
+  .control-label.col-md-3 {
+    text-align: left;
+  }
+
+  #wifi-fields. #wired-fields, #ntp-fields, #dns-fields {
+    display: none;
+  }
+
+  span.glyphicon:hover {
+    cursor: pointer;
+  }
+
+  #ntp-addbutton , #dns-addbutton, .remove-button  {
```

Comment:
If 3 elements are sharing these properties why not create a class `.app-btn` and apply that same **nameClass** on each element? I think this way the browser preforms less DOM parsing. The name is merely a suggestion.